.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: '/'
+    schedule:
+      interval: monthly
+    versioning-strategy: increase
+  - package-ecosystem: github-actions
+    directory: '/'
+    schedule:
+      interval: monthly
```

Comment:
Sometimes have these more frequently since they tend to be less noisy when you already have them pinned to major versions

Reply:
Agreed - good point